src/components/Recidiviz/RecidivizTable/RecidivizTable.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import RecidivizTable from './RecidivizTable';
+
+jest.mock('../../../lib/static/nlform.js');
+jest.mock('../../../lib/static/stateRecidivismRandom.js');
+
+it('renders the RecidivizTable component', () => {
```

Comment:
More test possibilities: checking the search functionality and asserting that the table only shows proper matches, asserting that the table is in descending order by recidivism rate, and checking that the contents of the table change accordingly by state.

Reply:
https://github.com/Recidiviz/recidiviz.com/pull/22#discussion_r186476649